app/dashboard/views.py

```diff
@@ -1578,6 +1579,28 @@ def get_users(request):
                 profile_json['avatar_url'] = user.avatar_url
             profile_json['preferred_payout_address'] = user.preferred_payout_address
             results.append(profile_json)
+        # try github
+        # TODO: rate limit? and/or use the logged in users github key instead of ours?
+        if not profiles:
+            search_results = search(q)
+            for result in search_results.get('items', []):            
+                profile_json = {}
+                profile_json['id'] = -1
+                profile_json['text'] = result['login']
+                profile_json['email'] = None
+                profile_json['avatar_id'] = None
+                profile_json['avatar_url'] = result['avatar_url']
+                profile_json['preferred_payout_address'] = None
+                results.append(profile_json)
+        # just take users word for it
+        if not profiles:
```

Comment:
should we check `if not profiles and not results` here to avoid the dupe if truthy? @owocki 
I'm about to commit a pygithub based search method to use our current token flow.  You'd just pass `request.access_token` / `access_token` from the context to: `search_users(query, token=access_token)` from `git.utils`

Reply:
lgtm